src/modules/private.ts

```diff
@@ -707,6 +710,68 @@ export default class Private {
     );
   }
 
+  /**
+   * @description get trading rewards for a user for a given epoch
+   *
+   * @param {
+   * @epoch number to request rewards data for
+   * }
+   */
+  getTradingRewards(
+    params: {
+      epoch?: number,
+    },
+    genericParams: GenericParams = {},
+  ): Promise<{ tradingRewards: TradingRewardsResponseObject }> {
+    return this._get(
+      'rewards/weight',
+      {
+        ...params,
+        ...genericParams,
+      },
+    );
+  }
+
+  /**
+   * @description get liquidity provider rewards for a user for a given epoch
+   *
+   * @param {
+   * @epoch number to request rewards data for
```

Comment:
no need to include `number` given that it's in the type

Reply:
done